src/test/java/org/example/controller/JobRolesControllerTest.java

```diff
@@ -47,27 +48,27 @@ void GetJobRoles_shouldReturnSuccessfulResponse_whenOpenJobRolesReturned() throw
         List<JobRoleResponse> jobRoleResponses = new ArrayList<>();
         jobRoleResponses.add(jobRoleResponse);
 
-        Mockito.when(jobRoleService.getAllJobRoles()).thenReturn(jobRoleResponses);
-        Response response = jobRoleController.getJobRoles();
+        Mockito.when(jobRoleService.getAllJobRoles(1,10)).thenReturn(jobRoleResponses);
+        Response response = jobRoleController.getJobRoles(1,10);
 
         assertEquals(200, response.getStatus());
     }
 
 
     @Test
     void getJobRoles_shouldReturnServerError_whenServiceFailsToGetJobRoles() throws
-            SQLException, DoesNotExistException {
-        Mockito.when(jobRoleService.getAllJobRoles()).thenThrow(SQLException.class);
-        Response response = jobRoleController.getJobRoles();
+            SQLException, DoesNotExistException, InvalidPageLimitException {
+        Mockito.when(jobRoleService.getAllJobRoles(1,10)).thenThrow(SQLException.class);
+        Response response = jobRoleController.getJobRoles(1,10);
 
         assertEquals(500, response.getStatus());
     }
 
```

Comment:
You have not tested the case where the InvalidPageLimitException is thrown. What scenario would this exception be thrown?